.ci/elastic_cloud_tf_gcp.yml

```diff
@@ -0,0 +1,42 @@
+---
+cluster_name: "{{ lookup('env','CLUSTER_NAME') }}"
+create_users: false
+grab_cluster_info: true
+create_ilm: false
+certificate_issuer: letsencrypt-staging
+
+elastic_cloud:
+  provider: gcp
+  region: gcp-us-central1
+  endpoint: https://staging.found.no
```

Comment:
@kuisathaverat , do you recommend to use this endpoint or the production one?

Somehow I see some flakiness in the staging environment and I don't know if that's related to the versions I use or the environment itself.

On the other hand, do you recall if versions with the format `8.4.0-abcdefgh-SNAPSHOT` are available in production? I cannot find them in `staging`. Those versions are the pinned versions for the snapshots that actually are bumped in https://github.com/elastic/e2e-testing/blob/main/.ci/bump-stack-version.sh. At the moment, I'm just using `8.4.0-SNAPSHOT` but if we could use the pinned versions then this will help from reproducibility.

Reply:
> this issue is probably not related to the environment.

Yes, it's not related to the docker image naming. Sorry If I mixed up two different questions.

Regarding the docker image name, I tried with [`8.4.0-42ce0eef-SNAPSHOT`](https://github.com/elastic/e2e-testing/blob/0c5991f9a4cc104ffa33019727cb65057272cabb/.ci/Jenkinsfile#L50) in staging and it did not work since it could not find the version. 

```
19:58:20  TASK [stack_tf : Create/update Elastic Cloud deployment] ***********************
19:58:20  Monday 20 June 2022  11:58:20 -0700 (0:00:01.044)       0:00:19.948 *********** 
19:58:26  fatal: [localhost]: FAILED! => changed=false 
...
19:58:26    msg: |-
19:58:26      Failure when executing Terraform command. Exited 1.
19:58:26      stdout: ec_deployment.main: Creating...
19:58:26    
19:58:26      stderr:
19:58:26      Error: failed creating deployment: 2 errors occurred:
19:58:26              * api error: clusters.cluster_invalid_plan: Version [8.4.0-42ce0eef-SNAPSHOT] is not available. Please use one of the available versions (resources.elasticsearch[0].elasticsearch.version)
```

Build logs -> [here](https://beats-ci.elastic.co/blue/organizations/jenkins/e2e-tests%2Fe2e-testing-macos-daily-mbp/detail/PR-2626/2/pipeline#step-84-log-221)